safe-global/safe-wallet-web
# [Seedless Onboarding] Adjust Account center design

## What it solves

Part of #2452

## How this PR fixes it

- Adds a "Enable multifactor authentication" Button to the account center if MFA is not enabled
- Adds a "Delete Account" Button if its not prod for easier testing
- Renames `WalletInfo` to `WalletOverview`
- Extracts part of `AccountCenter` into a new component called `WalletInfo` for better testability

## ToDos


## How to test it

1. Open a Safe
2. Connect with a social login for the first time
3. Open the Account center
4. Observe a new button to enable mfa that leads to the settings page
5. Enable MFA
6. Observe no longer seeing the button in the account center
7. Observe a Delete Account button if not on prod and logged in with social
8. Press the Button, reload the page and log in with social
9. Observe having a new wallet address and MFA is disabled

## Screenshots

<img width="340" alt="Screenshot 2023-10-20 at 13 22 01" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/d59a8de2-9584-44a0-a8c4-065e029edd12">


## Checklist
